plugins/dfu/fu-dfu-device.c

```diff
@@ -804,15 +804,21 @@ fu_dfu_device_request_detach(FuDfuDevice *self, FuProgress *progress, GError **e
 	FuDfuDevicePrivate *priv = GET_PRIVATE(self);
 	GUsbDevice *usb_device = fu_usb_device_get_dev(FU_USB_DEVICE(self));
 	const guint16 timeout_reset_ms = 1000;
+	guint8 force_detach = 0x00;
 	g_autoptr(GError) error_local = NULL;
 
+	if (fu_device_has_private_flag(FU_DEVICE(self), FU_DFU_DEVICE_FLAG_FORCE_DETACH)) {
+		g_debug("Force-Detach quirk");
```

Comment:
I don't think we need this g_debug line, it's already in the ->to_string() output.